src/styles/theme.ts

```diff
@@ -8,6 +8,7 @@ export type ThemeModeType = 'dark' | 'light'
 export interface ThemeAttributes {
   mode: ThemeModeType
   BACKGROUND: string
+  BOTTOM_SHEET_BACKGROUD: string
```

Comment:
What was it really necessary to add another attribute?

Reply:
Because isn't the same color than the main background

Reply:
And I do a spelling mistake to ^^

Reply:
Ok